pelican/plugins/seo/tests/conftest.py

```diff
@@ -193,6 +193,160 @@ def fake_article_multiple_elements():
     )
 
 
+@pytest.fixture()
```

Comment:
You don't need to add datas here to test the feature. As canonical feature plays with `metadata` object, you can add the metadata you need directly in the test, as [here](https://github.com/pelican-plugins/seo/blob/master/pelican/plugins/seo/tests/test_canonical_url_creator.py#L22).

Reply:
Done, moved to test_canonical_url_creator.py

Reply:
Seems good to me.